manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -355,6 +355,10 @@ data:
     # 99th percentile of the backend commit duration.
     - '{__name__="instance:etcd_disk_backend_commit_duration_seconds:histogram_quantile",quantile="0.99"}'
     #
+    # owners: (https://github.com/openshift/cluster-node-tuning-operator)
```

Comment:
Not sure a github repo should be a metric owner here. Can we find a team like for the other owner entries?

Reply:
Thanks Jan, for the review. There were some other owners pointing to a github repo in this file on the release-4.10 branch (from where it was backported). I will update it here to keep it more in line with the other entries.

Reply:
Thanks, I'll go after that other entry right away :wink: